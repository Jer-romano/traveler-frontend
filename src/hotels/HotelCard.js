import React from "react";

/** Show limited information about a hotel
 */

function HotelCard({ name, description, }) {
  console.debug("HotelCard");

  return (
    <div className="card mb-2">
        <div className="card-body">
          <h4 className="card-title">
            {name}
          </h4>
          <p>{description}</p>
        </div>
    </div>
  );
}

export default HotelCard;
