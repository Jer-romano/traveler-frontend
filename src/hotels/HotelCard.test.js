import React from "react";
import { render } from "@testing-library/react";
import HotelCard from "./HotelCard";
//TODO Add more tests!!

it("renders without crashing", function () {
    render(<HotelCard/>);
});

it("matches snapshot", function() {
    const { asFragment } = render(<HotelCard/>);
    expect(asFragment()).toMatchSnapshot();
});