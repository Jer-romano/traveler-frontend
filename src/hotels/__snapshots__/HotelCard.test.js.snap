// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card mb-2"
  >
    <div
      class="card-body"
    >
      <h4
        class="card-title"
      />
      <p />
    </div>
  </div>
</DocumentFragment>
`;
